doc/history.txt

```diff
@@ -43,6 +43,7 @@ Classes / members marked as obsolete
 Added/fixed
 ===========
 (x) #1302 Implement support for disposing of IDisposable ViewModels in WindowLogic
+(+) #1308 Add EventArgsConverter Support in EventToCommand
```

Comment:
Although this is an unwritten rule, new features go on top, then changes, then bug fixes, then removals. No need to change this for the PR, but I do want you to be aware for a potential next time :-)

Reply:
Whoops, that's what happens when I do that one last.